nuxt.config.ts

```diff
@@ -160,6 +160,7 @@ const config: NuxtConfig = {
       { code: 'ro', iso: 'ro-RO', name: 'Română', file: 'ro.json' },
       { code: 'sk', iso: 'sk-SK', name: 'Slovenčina', file: 'sk.json' },
       { code: 'sl', iso: 'sl-SI', name: 'Slovenščina', file: 'sl.json' },
+      { code: 'es', iso: 'es-ES', name: 'Español (España)', file: 'es.json' },
```

Comment:
Can we keep these somewhat alphabetical? It's a lot easier to figure out if one is added or not.

Reply:
@MrTimscampi done!